aten/src/ATen/Storage.h

```diff
@@ -6,8 +6,8 @@ namespace at {
 
 struct AT_API Storage {
 public:
-  Storage() = delete;
-  Storage(StorageImpl* storage_impl) : storage_impl_(storage_impl) {}
+  Storage() {}
```

Comment:
I forget if I asked about this -- do we actually need this?

Reply:
We don't have to give Storage a default constructor, but since it's a pointery thing, it seems more idiomatic C++ to give it one. Makes reasonable things work (like storing Storage in containers).

Reply:
Yah it's just...then we get back to the UndefinedTensor question, which I don't really want to deal with :).  But it's up to you.